front/app/components/Topics/UpdateFollowTopic.tsx

```diff
@@ -52,7 +76,7 @@ const UpdateFollowTopic = ({ topic }: Props) => {
   };
 
   return (
-    <Badge color={color} onClick={handleFollowOrUnfollow}>
+    <Badge color={color} onClick={handleFollowOrUnfollow} aria-busy={isLoading}>
```

Comment:
We are not setting aria-busy in the other place where we have the `setAnnouncement` `useEffect` btw.